src/ck-core/ckrdma.C

```diff
@@ -1250,19 +1251,7 @@ void CkRdmaEMBcastAckHandler(void *ack) {
 #if CMK_REG_REQUIRED
         deregisterMemFromMsg(myMsg, true);
 #endif
-        // send a message to the parent to signal completion
-        int srcPe;
         CkArray *mgr = NULL;
-        envelope *env = (envelope *)bcastInterimAckInfo->msg;
-        CkUnpackMessage(&env); // Unpack message before sending it to getParentBcastAckInfo
-        char *ref = (char *)(getParentBcastAckInfo(bcastInterimAckInfo->msg, srcPe));
-        CkPackMessage(&env);
-
-        NcpyBcastInterimAckInfo *ncpyBcastAck = (NcpyBcastInterimAckInfo *)ref;
-
-        // Invoke CkRdmaEMBcastAckHandler on my parent node
-        invokeRemoteNcpyAckHandler(ncpyBcastAck->origPe, ncpyBcastAck->parentBcastAckInfo, ncpyHandlerIdx::BCAST_ACK);
-
```

Comment:
Removal of sending ack message after the completion of all nodes below me in the tree.  (for bcast recv api).